src/sentry/static/sentry/app/components/charts/simpleTableChart.tsx

```diff
@@ -55,18 +56,22 @@ class SimpleTableChart extends React.Component<Props> {
           })}
         >
           {data.map((row, index) => this.renderRow(index, row, meta, columns))}
-        </PanelTable>
-      </TableWrapper>
+        </StyledPanelTable>
+      </React.Fragment>
     );
   }
 }
-const TableWrapper = styled('div')`
-  /* align height with charts */
-  height: 160px;
-  overflow-y: scroll;
 
-  /* Space away from where the widget title is */
-  margin-top: 40px;
+const StyledPanelTable = styled(PanelTable)`
+  border-radius: 0;
+  border-left: 0;
+  border-right: 0;
+  border-bottom: 0;
+
+  /* align height with charts */
+  height: 190px;
+  overflow: scroll;
+  margin: ${space(1)} 0 0;
```

Comment:
The table rows expand when there's only the header and one row.

Wondering if 

```
grid-template-rows: min-content;
```

will help here. 

<img width="638" alt="Screen Shot 2021-01-25 at 4 44 58 PM" src="https://user-images.githubusercontent.com/139499/105770097-c13afb80-5f2c-11eb-9ff2-217454b5712e.png">

Reply:
I'll give that a try.